test/ComponentsTest/AddChoreTest/AddChore.test.jsx

```diff
@@ -0,0 +1,44 @@
+import { describe, it, expect, vi } from "vitest";
+import { render, screen } from "@testing-library/react";
+import AddChore from "../../../src/components/AddChore/AddChore";
+import React from "react";
+import userEvent from "@testing-library/user-event";
+import "@testing-library/jest-dom/vitest";
+
+describe("AddChore", () => {
+  it("Checks if the AddChore method is being called (not updated)", async () => {
```

Comment:
Just to clarify, does this PR only cover the Add Chore modal then? Like the chore does not actually get added to the chore list yet?